db/schema.rb

```diff
@@ -10,15 +10,15 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_09_23_153522) do
```

Comment:
There are some oddities in the schema here, which will be resolved when we merge #379